src/test/resources/org/broadinstitute/hellbender/tools/picard/fasta/testfasta.fasta

```diff
@@ -0,0 +1,15 @@
+>Chromosome 20
```

Comment:
create a directory src/test/resources/org/broadinstitute/hellbender/tools/picard/fasta/NormalizeFastaIntegration
and move this file there.